UnityProject/Assets/Scripts/Items/Weapons/ActivatableWeaponComponents/Client/PreventHandswapWhileActive.cs

```diff
@@ -0,0 +1,53 @@
+namespace Items
```

Comment:
Namespaces are meant to reflect the folder/file structure, don't have two of them at once.